tests/integration/llm/client.py

```diff
@@ -975,7 +975,7 @@ def awscurl_run(data,
                 json_results=False,
                 random_delay=False):
     find_awscurl()
-    headers = "Content-type: application/json"
+    headers = "Content-type:application/json"
```

Comment:
Why this causing problem?

Reply:
See this example:

`./awscurl -c 1 -N 5 -X POST http://127.0.0.1:8080/invocations --connect-timeout 300 -H Content-type: application/json -d '[{"key": "what is panda?", "value": "hi"}]'   -P -t`

Having space in this the Content-type not passed in correctly. The other fix is add quote for Content-type.

Reply:
I see, we should add quote instead of remove space

Reply:
Changed.